### PR TITLE
ARM idns option added in NIC create

### PR DESCRIPTION
**Pull request by sreekanthbs**
ARM idns option added in NIC create

### CODE REVIEW

File: test/commands/arm/network/arm.network.nic-tests.js [hunk 1]
```diff
@@ -110,8 +109,8 @@ describe('arm', function() {
                             networkUtil.showLB(groupName, LBName, suite, function(result) {
                               networkUtil.createNSG(groupName, nsgName, location, suite, function(result) {
                                 networkUtil.showNSG(groupName, nsgName, suite, function(result) {
-                                  var cmd = util.format('network nic create %s %s %s -t priority=low -u %s -k %s -m %s -w %s -o %s -a %s -d %s -e %s --json',
-                                    groupName, nicPrefix, location, networkTestUtil.subnetId, subnetprefix, vnetPrefix, networkTestUtil.nsgId, nsgName, privateIP, networkTestUtil.lbaddresspoolId, networkTestUtil.lbinboundruleId).split(' ');
+                                  var cmd = util.format('network nic create %s %s %s -t priority=low -u %s -k %s -m %s -w %s -o %s -a %s -d %s -e %s -r %s -f %s --json',
+                                    groupName, nicPrefix, location, networkTestUtil.subnetId, subnetprefix, vnetPrefix, networkTestUtil.nsgId, nsgName, privateIP, networkTestUtil.lbaddresspoolId, networkTestUtil.lbinboundruleId, internalDnsLable, enableIpForwarding).split(' ');
```
Comment (babumuralidharan) [inline test/commands/arm/network/arm.network.nic-tests.js]: indent check

File: test/commands/arm/network/arm.network.nic-tests.js [hunk 2]
```diff
@@ -132,7 +131,7 @@ describe('arm', function() {
       });
 
       it('set should modify nic', function(done) {
-        var cmd = util.format('network nic set %s %s -t priority=high -w %s -o %s -a %s -u %s -k %s -d %s -e %s --no-tags --json', groupName, nicPrefix, networkTestUtil.nsgId, 'NoSuchNSGExists', privateIP2, networkTestUtil.subnetId, 'NoSuchSubnetExists', networkTestUtil.lbaddresspoolId, networkTestUtil.lbinboundruleId).split(' ');
+        var cmd = util.format('network nic set %s %s -t priority=high -w %s -o %s -a %s -u %s -k %s -d %s -e %s --no-tags -r %s -f %s --json', groupName, nicPrefix, networkTestUtil.nsgId, 'NoSuchNSGExists', privateIP2, networkTestUtil.subnetId, 'NoSuchSubnetExists', networkTestUtil.lbaddresspoolId, networkTestUtil.lbinboundruleId, internalDnsLableN, enableIpForwardingN).split(' ');
         testUtils.executeCommand(suite, retry, cmd, function(result) {
```
Comment (babumuralidharan) [inline test/commands/arm/network/arm.network.nic-tests.js]: have the same priority value reflected here

File: test/commands/arm/network/arm.network.nic-tests.js (hunk 1 above)
Comment (sreekanthbs) [inline test/commands/arm/network/arm.network.nic-tests.js]: Done

File: test/commands/arm/network/arm.network.nic-tests.js (hunk 2 above)
Comment (sreekanthbs) [inline test/commands/arm/network/arm.network.nic-tests.js]: done

**Pull request merged**